docs/preview/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Home"
+layout: default
+permalink: /
+redirect_from:
+ - /index.html
+---
+
+# Installation
+
+## Web API
+
+```shell
+PM > dotnet new --install Arcus.Templates.WebApi
+```
+
+Read [here](features/web-api-template) for standard and configurable features.
+
+## Azure Service Bus Queue
+
+```shell
+PM > dotnet new --install Arcus.Templates.ServiceBus.Queue
+```
+
+Read [here](features/servicebus-queue-worker-template) for standard and configurable features.
+
+## Azure Service Bus Topic
+
+```shell
+PM > dotnet new --install Arcus.Templates.ServiceBus.Topic
+```
+
+Read [here](features/servicebus-topic-worker-template) for standard and configurable features.
+
+## Azure Functions Databricks
+
+```shell
+PM > dotnet new --install Arcus.Templates.AzureFunctions.Databricks
```

Comment:
```suggestion
PM > dotnet new --install Arcus.Templates.AzureFunctions.Databricks.JobMetrics
```

Reply:
So you want another namespace and project alias too?

Reply:
I'm just thinking long-term in case we provide other templates but if it's too much work then it's ok

Reply:
It's some work, but can be done.

Reply:
Done, resolving.